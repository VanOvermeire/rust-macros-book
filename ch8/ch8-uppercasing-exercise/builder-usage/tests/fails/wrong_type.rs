use builder_macro::Builder;

#[derive(Builder)]
struct Gleipnir {
    #[uppercase]
    roots_of: i32,
}

fn main() {}
