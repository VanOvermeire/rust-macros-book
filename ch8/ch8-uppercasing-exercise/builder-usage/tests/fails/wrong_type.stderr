error: Can only use uppercase for String type
 --> tests/fails/wrong_type.rs:6:5
  |
6 |     roots_of: i32,
  |     ^^^^^^^^
